### ISSUE TITLE
Automatic reset

### ISSUE THREAD

**Issue by kartheek7895** [martin31821/cpupower#66]
No matter the profile set, the min. freq, max. freq and turbo boost values are resetting to some default value (11%,100%). 

OS :`ubuntu 18.04`

Edit: Tried using `cpupower` directly, and it is working fine with root privileges, I'm guessing there might be a root privileges issue with the extension.

**Comment by martin31821** [comment]
Usually, the extension tries to detect whether policykit rules are installed, you can try to fiddle around with the `installer.sh` script to install the p11kit rule.

**Comment by kartheek7895** [comment]
```
./installer.sh install
Installing policykit action... ./installer.sh: line 56: /usr/share/polkit-1/actions/mko.cpupower.setcpufreq.policy: Permission denied
```
```
sudo ./installer.sh install
Installing policykit action... Success
Fixing permissions... Success
```

Still, the issue persists. Any idea what might be happening?

**Comment by martin31821** [comment]
I'll setup an ubuntu machine to verify the problem.

**Comment by claford-v-lawrence** [comment]
It seems that the version on the official gnome extension site is a bit outdated. I run ubuntu 18.04 on a thinkpad and it is not responding either.

**Comment by fin-ger** [comment]
Ubuntu 18.04 was tested and issue will be fixed in v9.0.0
